palace/drivers/basesolver.hpp

```diff
@@ -68,12 +67,22 @@ class BaseSolver
                          double time) const;
   void PostprocessFields(const PostOperator &postop, int step, double time) const;
 
+  // Postprocess granular error indicator to file. The argument normalized indicates if
+  // supplied indicators have already been normalized.
+  void PostprocessErrorIndicators(const std::string &name, int step, double time,
```

Comment:
Without assigning a special value meaning to "step", there isn't enough information entering to differentiate a row led by `time` vs a string value `"Mean"`. Another option to the overload is an extra argument to ignore the step value, but I opted against that as it makes both calls more complicated compared to having the compiler pick between the overloads.

I want it to report the granular estimates given we are calculating them anyway (for instance to say which mode or frequency is contributing most to a given adaptation), but also want to report the quantity used to drive the adaptation. In the revision there are only doubles going in, to avoid passing a communicator, so there's even less information.

Reply:
Seems like the simplification suggested above, of just not reporting the mean, keeps the structure of the output files the same and resolves this issue.

Reply:
It also means that all the log files never actually contain information about the actual combined quantity being used to drive the adaptation. I can make the change, but I strongly disagree with it if the only goal is avoiding that overload.

Reply:
It can still be printed to the log, but doesn't need to be in the postprocessing files where the user is going to have access to the same abilities to compute it as a derived quantity.

For the log output, I'd recommend not including it in this PR and just managing the printing as part of the SEMR loop.

Reply:
https://github.com/awslabs/palace/pull/85/commits/fc5000601a6bd4ede4cfe331884a67c37bcb7054